### DISCUSSION TITLE
BB DB - Update Row Automation - Date Time Attribute Existing Value

### DISCUSSION THREAD

**Discussion by Brrrradley** [Budibase/budibase#4903] [Help] [answered]
I am working on a prototype application using a BB internal db (with several date-time attributes), and have several update row automations configured.  I cannot seem to be able to retrieve the existing value of a date-time attribute via handlebars (ie: {{trigger.row.DateAttribute}} ) as clicking into the field pulls up a date-picker.  Leaving as blank overwrites the previous value with a null, which is undesired behavior.   

How do I retrieve the existing value of a date-time attribute in an update automation step?

**Comment by mjashanks** [comment, marked as answer]
Hey @Brrrradley - select the "Use Bindings" option:

![image](https://user-images.githubusercontent.com/3524181/158225029-007f0dce-7975-4efb-9a09-c39cefa2a33f.png)

**Reply by Brrrradley** [reply]
I don't even see that as an option.  

Using self-hosted version 1.0.73
![BB](https://user-images.githubusercontent.com/101414789/158227274-c748e77e-46bb-4d70-9577-5f293981155b.png)

**Reply by mjashanks** [reply]
Ah, I am using 1.0.86 - this has been added very recently
